src_py/sysfont.py

```diff
@@ -138,19 +139,80 @@ def initsysfonts_win32():
     return fonts
 
 
-def initsysfonts_darwin():
-    """read the fonts on OS X. X11 is required for this to work."""
-    # if the X11 binary exists... try and use that.
-    #  Not likely to be there on pre 10.4.x ...
-    if exists("/usr/X11/bin/fc-list"):
-        fonts = initsysfonts_unix("/usr/X11/bin/fc-list")
-    # This fc-list path will work with the X11 from the OS X 10.3 installation
-    # disc
-    elif exists("/usr/X11R6/bin/fc-list"):
-        fonts = initsysfonts_unix("/usr/X11R6/bin/fc-list")
-    else:
-        fonts = {}
+def fetch_elements_from_xmlString(xml_string, Xpath):
+    """Gets the specified element iterable"""
+
+    #Get the ElementTree object from the xml string
+    mac_fonts_xml_tree = ET.fromstring(xml_string)
+
+    #Get the elements of the tree where the font information is located
+    font_xml_elements = mac_fonts_xml_tree.iterfind(Xpath)
+
+    return font_xml_elements 
+
+def textContentList_from_xmlElement(element):
+    """Gets the content of tags (returned as a list) from the specified element and its subelements"""
+    #Gets all the text within each tag in the node
+    tag_text = element.itertext()
+
+    #Cleans the list of any whitespace items
+    tag_content_list = [content for content in tag_text if content.strip() != ""]
+
+    return tag_content_list
+
```

Comment:
maybe we can also do this in one line ?
```python
def textContentList_from_xmlElement(element)
     return [c for c in element.itertext() if c.strip()]
```

Reply:
@e1000  
Thank you for the suggestions, I made the adjustments :D